### PR TITLE
Feature/built value split

### PR DESCRIPTION
**Pull request by Leptopoda**
the `nextcloud.openapi.g.dart` is not included as the build will currently fail :)

But at least it should be fairly easy to review

### CODE REVIEW

**Comment by Leptopoda** [comment]
Yeah tests are abviously expected to fail

File: packages/dynamite/pubspec.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ environment:
 
 dependencies:
   build: ^2.3.1
+  built_value: ^8.4.4
```
Comment (provokateurin) [inline packages/dynamite/pubspec.yaml:9]: This isn't used, is it?

**Review by provokateurin — CHANGES_REQUESTED** [review]

File: packages/dynamite/pubspec.yaml (hunk 1 above)
Comment (Leptopoda) [reply packages/dynamite/pubspec.yaml:9]: I thought I need it for the build.yaml file but turns out I don't

**Comment by Leptopoda** [comment]
Also I noticed the imports are missing in the generated file.

**Comment by Leptopoda** [comment]
I also removed the json_serializable import as we are not depending on them anymore.

I didn't bother keeping it for now :)

File: packages/nextcloud/lib/src/nextcloud.openapi.g.dart
Comment (provokateurin) [inline packages/nextcloud/lib/src/nextcloud.openapi.g.dart:1]: Maybe remove this file in a separate commit before doing any changes? Idk when you will add it back

**Review by provokateurin — APPROVED** [review]

File: packages/nextcloud/lib/src/nextcloud.openapi.g.dart
Comment (Leptopoda) [reply packages/nextcloud/lib/src/nextcloud.openapi.g.dart:1]: Adding the imports lets the generator run them so It is back in my latest push.

It got auto removed as the generator failed without the imports.

**Pull request merged**